sdk/nodejs/index.ts

```diff
@@ -28,7 +28,8 @@ import * as dynamic from "./dynamic";
 import * as iterable from "./iterable";
 import * as log from "./log";
 import * as runtime from "./runtime";
-export { asset, dynamic, iterable, log, runtime };
+import * as service from "./service";
```

Comment:
Is `service` the right name here?

Alternatives might be:
1. `pulumi.getStack()`
2. `pulumi.backend.getStack()`
3. `pulumi.provider.getStack()`
4. `pulumi.Stack.get()`

Not sure any of those are better....

Reply:
I couldn't think of a better name:
1. `pulumi.getStack()` is already taken :(
2, 3. I don't mind either of these. I have a slight preference for `backend`.
4. `pulumi.Stack.get` would do something ~different than our other `get`s. If we can convince ourselves that this is okay, I think that this is a pretty good option.